app/src/main/res/values/dimens.xml

```diff
@@ -1,3 +1,11 @@
 <resources>
     <dimen name="fab_margin">16dp</dimen>
+    <dimen name="margin_x4">16dp</dimen>
```

Comment:
Aha, here I see x<Y> convention. First time I see it like this. What is the idea to have 4dp grunuality?

Reply:
What if I want to have padding 16dp? Should I reuse this constant or create a new one?

Reply:
Actually that graduality was only dictated by Figma margins. Do you suggest to generify dimens naming?

Reply:
That what we do with the current project but let me ask other mentors